package.json

```diff
@@ -27,8 +27,10 @@
     "iconv-lite": "0.4.13",
     "minimist": "^1.2.0",
     "native-keymap": "0.1.2",
+    "pty.js": "git+https://github.com/jeremyramin/pty.js.git#28f2667",
     "sax": "1.1.2",
```

Comment:
https://github.com/Microsoft/vscode/issues/6353 and the "Cannot find module '../build/Release/pty.node'" warning is related to this. The prebuilt packages are used because the build fails so the warning shows up. Eventually we will probably want to fork pty.js ourselves to sort out any dependency problems.